content/practices/research-workshop/index.md

```diff
@@ -0,0 +1,128 @@
+   - Record aspects of the session that matter (e.g. audio, video, etc.)
+   
+   Synthesize Findings
+   - All types: group like comments across users to find trends
+   - Usability only: create pass/fail table to quickly see which aspects of the design worked and which did not
+   - See Research Synthesis for more information
+   
+   {{< callout >}}
+   Tip: Drawing these four steps on the board showing a progression can be really helpful to visualize how they flow together
+   {{< /callout >}}
+   
+1. Share **biases** to be aware of and key **“dos”** and **“don’ts”** when conducting research by printing out or projecting this **Research Dos & Don'ts Guide** 
+
+  ***EDITOR NOTE: The above Research Dos & Don'ts Guide is an internal google doc. Please change, publish the doc, or find an alternative***
+
+1. Finally, spend the remaining time **role playing** to get a feel for what actual research will be like:
+   
+   Option 1: Pair Role Play
+   - Break into pairs, giving each person a “Pause” card
+   - Identify who will be playing each role (Interviewee, Interviewer)
+   - Interviewer uses one of the team’s discussion guides to ask questions of Interviewee
+   - Conduct the interview for 10 minutes, pausing whenever one of the pairs shows the “Pause” card to discuss specific feedback
+   - Reflect on the experience using a plus/delta format
+   
+   Option 2: Group Role Play
+   - Identify who will be playing each role (Interviewee, Note-takers, Interviewer)
+   - Interviewer uses one of the team’s discussion guides to ask questions of Interviewee while Note-takers take notes that, to the extent possible, mimic “real” notes
+   - Conduct the interview for 10 minutes
+   - Reflect on the experience using a plus/delta format
+
+### Success/Expected Outcomes
+You’ve succeeded when the team feels like they’re confident they can conduct research in a way that the team can garner usable information. 
+
+## Facilitator Notes & Tips
+You can make as many or as few visual aids ahead of time as you like to support this workshop, depending on your preference. It can be equally successful printing/hanging things you want to focus on as it is drawing them as you talk.
+
+If the team feels like they want more practice conducting research before they do it with actual users, don’t hesitate to schedule a follow-up session or two. Role playing can do wonders to prep people who haven’t done a lot of research before but are excited to learn how.
+
+## Related Practices
+
+### Variations
+None at the moment. 
+
+### Preceding
+None at the moment. 
+ 
+### Following
+None at the moment. 
+
+## Real World Examples
+
+![Research Workshop Example 1](/images/practices/research-workshop/example-1.jpg)
+
+![Research Workshop Example 2](/images/practices/research-workshop/example-2.jpg)
+
+## Recommended Reading
+None at the moment
```

Comment:
Recommended reading:
Lean UX (https://www.oreilly.com/library/view/lean-ux/9781449366834/) by Jeff Gothelf and Josh Seiden
UX For Lean Startup (https://www.amazon.com/UX-Lean-Startups-Experience-Research/dp/1449334911) by Eric Ries

Reply:
Added